ci/dhis2metadata-publish.yml

```
@@ -0,0 +1,40 @@
+name: 'dhis2-metadata: publish package'
+
+on:
+  push:
+    tags:
+      - '**'
```

I think this would be a shorter equivalent:

```suggestion
    tags:
```